miracl/flow/miracl_workflow_ace_parser.py

```diff
@@ -410,7 +410,7 @@ def parsefn(self) -> argparse.ArgumentParser:
             "--rca_side",
             type=str,
             choices=["rh", "lh"],
-            default="rh",
+            default=None,
```

Comment:
What would happen if we set all default values in our ACE parser to `None`? We would have to add `!= None` checks to all the module parsers. Would that make sense?

Reply:
Yeah, exactly that's what we would have to do. Some parameters for module functions are allowed to be `None` (for example registration checks if the `side` is `None`). But other module functions do not check it, so it would give an error because it would try to use the string `"None"` as the value for some parameter. 

So the way to handle it outside of the ACE code would be to go through all other functions and make sure they will check if the arg is `None` before using it.

Reply:
Should we do that before we merge into `main`...............? Let's discuss offline?